Formula/gcc.rb

```diff
@@ -232,6 +232,16 @@ def post_install
 
       EOS
       inreplace(specs, " %o ", "\\0%(homebrew_rpath) ")
+    elsif OS.mac?
+      # libstdc++: Fix iostream init for Clang on Darwin
+      # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=110432#c7
+      # https://gcc.gnu.org/git/gitweb.cgi?p=gcc.git;h=fe2651affa8c15624188bfd062fb894648743431
+      if version >= "13.2.0"
+        odie "Review if the fix is still required."
+      else
+        inreplace "/usr/local/opt/gcc@13/include/c++/13/iostream",
```

Comment:
We shouldn't hardcode `/usr/local` here, as that isn't the only prefix that we support.

Reply:
`#{HOMEBREW_PREFIX}` would be the correct replacement for it?